src/batch.py

```diff
@@ -203,39 +204,125 @@ def __parse_score_type(self, s):
             stype = 4
 
-            if m:
-                thresh = m.group(1) + ',' + m.group(2)
+            if valid:
+                thresh = match.group(1) + ',' + match.group(2)
             else:
-                thresh = '-0.75,0.75'
+                thresh = ''
 
                 self.warns.append(
-                    'Invalid threshold. Using -0.75 < Correlation < 0.75'
+                    'Invalid threshold. No default threshold set for correlation.'
                 ) 
 
         elif s.lower().find('effect') != -1:
-            ## Same comments as the correlation regex
-            m = re.search(r"([0-9.-]{2,})[^0-9.-]*([0-9.-]{2,})", s.lower())
+
+            valid, match = self.validate_effect_range(s.lower())
             stype = 5
 
-            if m:
-                thresh = m.group(1) + ',' + m.group(2)
+            if valid:
+                thresh = match.group(1) + ',' + match.group(2)
             else:
-                thresh = '0,1'
-                self.warns.append('Invalid threshold. Using 0 < Effect < 1.')
+                thresh = ''
+                self.warns.append('Invalid threshold. No default threshold set for effect.')
 
         else:
             self.errors.append('An unknown score type (%s) was provided.' % s)
 
         return (stype, thresh)
 
+    @staticmethod
+    def validate_correlation_range(value: str) -> Tuple[bool, Union[Match, None]]:
+        """
+        Validates whether the given string matches the format of a correlation range.
+
+        The correlation range is expected to be in the format:
+        `<lower_bound> < correlation < <upper_bound>`, where:
+        - `<lower_bound>` and `<upper_bound>` are numeric values (integer or decimal).
+        - `Correlation` is a keyword separating the bounds.
+        - `<` is the comparison operator.
+
+        Arguments:
+            value (str): The string to validate.
+
+        Returns:
+            tuple:
+                - bool: True if the range is valid and the lower bound is less than the upper bound, False otherwise.
+                - re.Match | None: The match object if the string matches the format, None otherwise.
+
+        Example:
+            validate_correlation_range("-0.75 < correlation < 0.75") -> (True, <Match object>)
+            validate_correlation_range("1 < correlation < -1") -> (False, None)
+        """
+
+        # Regex pattern for validating correlation range (e.g., -0.75 < Correlation < 0.75)
+        # Explanation:
+        # 1. `^-?` allows an optional negative sign for the first number.
+        # 2. `[0-9]+(\.[0-9]+)?` matches integers or decimals.
+        # 3. `<` ensures the presence of the comparison operator.
+        # 4. `-?` allows an optional negative sign for the second number.
+        # 5. `[0-9]+(\.[0-9]+)?` matches the second numeric value.
+        correlation_range_regex = r"^(-?[0-9]+(\.[0-9]+)?)\s*<\s*correlation\s*<\s*(-?[0-9]+(\.[0-9]+)?)$"
```

Comment:
I really like that you document what this regex does.  My regex is years old and really rusty, so it's nice to see an explanation.